tests/functional/testBenchmark.js

```diff
@@ -225,4 +225,225 @@ describe('Benchmark', function() {
     // Exit the benchmark
     await vpn.waitForQueryAndClick(queries.screenHome.CONNECTION_INFO_TOGGLE);
   });
+
+  describe('speed tests related telemetry tests', () => {
+    if(vpn.runningOnWasm()) {
+      // No Glean on WASM.
+      return;
+    }
+
+    // Telemetry design is detailed at:
+    // https://miro.com/app/board/uXjVM_QZzjA=/?share_link_id=616399368132
+
+    it("records events on succesful speed test", async () => {
```

Comment:
```suggestion
    it("records events on successful speed test", async () => {
```
Sorry, I only half-fixed the typo before. 🤦🏻